sdk/iot/README.md

```diff
@@ -1,14 +1,38 @@
 
 ## Getting started
 
-TBD
+- **Docs**: If you're new to the Azure IoT Embedded C SDK take a look at the [Machine State diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities. 
+
+- **Build**: The Azure IoT library is compiled following the same steps listed on the root [README](https://github.com/Azure/azure-sdk-for-c/blob/master/README.md) documentation, under ["Getting Started Using the SDK"](https://github.com/Azure/azure-sdk-for-c/blob/master/README.md#getting-started-using-the-sdk). 
+
+
+- **Samples**: [This page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/hub/samples) explains samples for the Azure Embedded C SDK IoT Hub and [this page](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/iot/provisioning/samples) shows the Provisioning Clients and how to use them.
 
 ### Prerequisites
 
-TBD
+For compiling the Azure IoT Embedded C SDK for the most common platforms (Windows and Linux), no further prerequisites are necessary.
+Please follow the instructions in the Getting Started section above.
+For compiling for specific target devices, please refer to their specific toolchain documentation. 
+
+## Key Features
+
+
+Feature | Azure IoT Embedded C SDK | Description
+---------|----------|---------------------
+ [Send device-to-cloud message](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-d2c) | [Suported] | Send device-to-cloud messages to IoT Hub with the option to add custom message properties. 
+ [Receive cloud-to-device messages](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-c2d) | [Supported] | Receive cloud-to-device messages and associated properties from IoT Hub.   
+ [Device Twins](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins) | [Supported] | IoT Hub persists a device twin for each device that you connect to IoT Hub.  The device can perform operations like get twin document, subscribe to desired property updates.
+ [Direct Methods](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-direct-methods) | [Supported] | IoT Hub gives you the ability to invoke direct methods on devices from the cloud.  
+ [DPS - Device Provisioning Service](https://docs.microsoft.com/azure/iot-dps/) | [Supported] | This SDK supports connecting your device to the Device Provisioning Service via, for example, [individual enrollment](https://docs.microsoft.com/azure/iot-dps/concepts-service#enrollment) using a [X.509 leaf certificate](https://docs.microsoft.com/azure/iot-dps/concepts-security#leaf-certificate).  
+ Protocol | MQTT | The Azure IoT Embedded C SDK suports only MQTT.  
+ Retry Policies | [Partially supported] | The Azure IoT Embedded C SDK provides guidelines for retries, but actual retries should be handled by the application.

```

Comment:
Fix naming, see comment further down.

Reply:
I would remove this part completely: since our clients don't manage any networking, there is not possibility for retries.
Most MQTT stacks already have automatic retries for QoS1 as well as automatic reconnect.